website/vcd.erb

```diff
@@ -564,6 +567,9 @@
             <li<%= sidebar_current("docs-vcd-resource-org-saml") %>>
               <a href="/docs/providers/vcd/r/org_saml.html">vcd_org_saml</a>
             </li>
```

Comment:
Missing link to `vcd_api_token` resource - website/docs/r/api_token.html.markdown

Reply:
Thanks for spotting, fixed.